assets/src/domain/eventEditor/ui/datetimes/dateForm/multiStep/ContentBody.tsx

```diff
@@ -33,29 +32,33 @@ const ContentBody: React.FC = ({ children }) => {
 					<div>
 						<DateFormSteps current={current} />
 						{/* RFF fields */}
-						{current === 0 && children}
+						{current === 0 && (
+							<>
+								{children}
+								<ButtonRow rightAligned>
```

Comment:
doesn't really make sense to have a button row with a single button, had to do this to align that button on the right side

Reply:
> doesn't really make sense to have a button row with a single button

why? it's just a layout element that separates the buttons from the other elements on the page.

How does the `rightAligned` prop affect things with regards to RTL ?

Reply:
I've used `justify-content: flex-end`, seems ok on RTL:

![image](https://user-images.githubusercontent.com/1477453/83650640-3e41eb00-a5c1-11ea-9596-9890f6d97331.png)